parcel.py

```diff
@@ -587,7 +588,7 @@ def new_report():
         wh = get_warehouse()
         form = flask.request.form.to_dict()
         metadata = {k: form.get(k, '') for k in REPORT_METADATA}
-        data_map = zip(['country', 'category'], [COUNTRIES, CATEGORIES])
+        data_map = zip(['lot', 'product'], [LOTS, COUNTRY_LOT_PRODUCTS.get(metadata['lot'], ())])
```

Comment:
`['lot', 'product']` is `REPORT_METADATA`, right? Use the variable rather than a hardcoded value.